integration/sql/iface-java/src/java/io/openlineage/sql/SqlMeta.java

```diff
@@ -0,0 +1,47 @@
+package io.openlineage.sql;
+
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class SqlMeta {
+    private final List<DbTableMeta> inTables;
```

Comment:
There doesn't appear to be a public way of getting access to the input / output tables.

Shouldn't we have some getter or setter?

Reply:
huh, weird, I had them locally. Maybe I accidentally deleted them resolving some conflict.
In any way, yes, they should absolutely be there.